tests/test_port_dpb_system.py

```diff
@@ -0,0 +1,851 @@
+from swsscommon import swsscommon
+import redis
+import time
+import os
+import pytest
+from pytest import *
```

Comment:
I don't think this is needed either - all the marks + whatnot should be covered by the `import pytest`.